api/node/models/models.go

```diff
@@ -36,7 +36,7 @@ func ParseHash32(h Bytes20) (types.Hash32, error) {
 
 func ParseBeacon(b Beacon) (types.Beacon, error) {
 	if len(b) != len(types.Beacon{}) {
-		return types.Beacon{}, fmt.Errorf("beacon length must be 8 (was: %d)", len(b))
+		return types.Beacon{}, fmt.Errorf("beacon length must be 4 (was: %d)", len(b))
 	}
```

Comment:
NIT: Could be dynamic instead of hard coded (in case we change beacon size at some point):

```go
if len(b) != len(types.EmptyBeacon) {
    return types.EmptyBeacon, fmt.Errorf("beacon length must be %d (was: %d)", len(types.EmptyBeacon), len(b))
}